handbook/engineering/deployments.md

```diff
@@ -116,35 +116,42 @@ These example commands are for the `dot-com` cluster where the Sourcegraph appli
 </tr>
 
 <tr>
-  <td>Edit a "deployment" (such as changing environment variables).</td>
+  <td>Edit a "deployment" (such as changing environment variables)</td>
   <td><code>kubectl edit deployment --namespace=prod DEPLOYMENT_NAME</code><br/>
   Note that the deployment name is not the pod name, and affects all pods running that deployment.</td>
 </tr>
 
 <tr>
   <td>SSH into the VM running a pod</td>
-  <td>Find the node ID from the NODE column of <code>kubectl get pods --namespace=prod -o=wide</code>. Go to the Google Compute Engine dashboard and click the "SSH" button in the top left to get the <code>gcloud</code> command to SSH into the node.<br /><code>kubectl -n prod exec -it POD_NAME /bin/sh</code></td>
+  <td>Find the node ID from the NODE column of <code>kubectl get pods --namespace=prod -o=wide</code>. Go to the Google Compute Engine dashboard and click the "SSH" button in the top left to get the <code>gcloud</code> command to SSH into the node<br /><code>kubectl -n prod exec -it POD_NAME /bin/sh</code></td>
```

Comment:
Can you add back the periods? Anything that is a complete sentence should end with a period, and we should prefer to use complete sentences everywhere.